src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -221,6 +223,15 @@ internal void Show(Guide.GuideFlow currentFlow)
             //After UI Automation and calculate the target we need to highlight the element (otherwise probably won't exist)
             SetHighlightSection(true);
 
+            if (HostPopupInfo.WindowName != null  && HostPopupInfo.WindowName.Equals(nameof(LibraryView)))
+            {
+                var automationStep = (from automation in UIAutomation
+                                      where automation.Name.Equals("CalculateLibraryItemLocation")
```

Comment:
Can you add these strings as constants as they are used in multiple locations

Reply:
@reddyashish @QilongTang this hard-coded text was changed to be a const string, see changes in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12394/commits/73e325c129f4636211e601c1d9ce9bf54abf255c